pkg/controller/install/install_controller.go

```diff
@@ -7,6 +7,7 @@ import (
 	mf "github.com/jcrossley3/manifestival"
 	eventingv1alpha1 "github.com/openshift-knative/knative-eventing-operator/pkg/apis/eventing/v1alpha1"
 	"github.com/openshift-knative/knative-eventing-operator/version"
+	configv1 "github.com/openshift/api/config/v1"
```

Comment:
you only need this if you're querying openshift-specific resources. you may eventually, but i would remove it until you do.